docs/install.md

```diff
@@ -10,15 +7,34 @@ Only **Linux** platforms are supported now. These instructions have been tested
 ## Prerequisites
 * **Erlang OTP** `>=20` (check with `elixir --version`)
 * **Elixir** `>=1.6` (check with `elixir --version`)
-* **solc** `>=0.4.24` (check with `solc --version`)
+* **solc** `~>0.4.24` (check with `solc --version`)
```

Comment:
Should we have this match with the latter instructions grabbing [0.4.26](https://github.com/omisego/elixir-omg/pull/773/files#diff-64407d3e6d5ef382426bdfbcae040c6aR76)? 

Also FWIW, the contracts seem to mix minor versions from [0.4.0](https://github.com/omisego/plasma-contracts/blob/master/contracts/ByteUtils.sol), [0.4.18](https://github.com/omisego/plasma-contracts/blob/master/contracts/MintableToken.sol). so maybe relax this to `~>0.4`?

Reply:
Good call, updated in https://github.com/omisego/elixir-omg/pull/773/commits/4592b31f27dfe200d1aaec2705841d8d3426baee